spec/examples/record/record_set_spec.rb

```diff
@@ -811,6 +811,46 @@ def self.latest(count)
     end
   end
 
+  describe '#page_size' do
+    it 'should perform query with specified page_size' do
+      expect_query_with_options(/SELECT/, :page_size => 1) do
+        Post.page_size(1).to_a
+      end
+    end
+
+    it 'should return the number of records specified by page_size' do
+      expect(Post.page_size(2).to_a.length).to be(2)
+    end
+  end
+
+  describe '#next_paging_state' do
+    it 'should return the paging state of the result' do
+      expect(Post.page_size(1).next_paging_state).not_to eq(nil)
+    end
+  end
+
+  describe '#paging_state' do
+    let(:page_one) { Post.page_size(1) }
+    let(:page_two) { Post.page_size(1).paging_state(page_one.next_paging_state) }
```

Comment:
It depends on the structure of the table, but the one returned by page_one looks like this:

`"\x00\tcassandra\x00\x12\x00\acequel0\x00\x00\x05title\x00\x7F\xFF\xFF\xFE"`

Since it contains data about the table, the documentation for the ruby driver cautions that it should be encrypted:

http://www.rubydoc.info/gems/cassandra-driver/2.0.0/Cassandra/Execution/Options:paging_state